frontendservice/src/views/specificShop.vue

```diff
@@ -142,8 +143,22 @@ export default {
             });
           }
         });
+        self.getWaitingTime();
+        if (this.isTokenInCookie) {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.tokeninfo.customersAhead;
+        } else {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.shopinfo.customersInQueue;
+        }
+    },
+    getWaitingTime() {
```

Comment:
getTotalWaitingTime?

Reply:
Changed to getWaitingTimePerCustomer() as discussed.